lib/house.rb

```diff
@@ -0,0 +1,47 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @above_market_price = "$500,000"
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def add_commas_to_price(price)
+    @price.to_s.reverse.gsub(/(\d{3})(?=\d)/, '\\1,').reverse
+  end
+
+  def above_market_price?
+    @price > @above_market_price
```

Comment:
your `@price` and your `@above_market_price` are both strings here... this technically works, but it's more of a happy accident.  Be sure that if you're comparing two values, that you're comparing them as integers, not as strings, since comparing strings can end up with some unexpected behavior.

Reply:
the idea behind the logic is very good though!